create-or-update-files.js

```diff
@@ -100,26 +88,20 @@ module.exports = function(octokit, opts) {
           });
         }
 
+        if (Array.isArray(change.filesToDelete) && change.filesToDelete.length > 0) {
+          await handleFilesToDelete(octokit, change, owner, repo, branchName, reject, treeItems);
```

Comment:
If the promise is rejected in `handleFilesToDelete` then execution doesn't stop here. The overall promise is rejected, but we'd continue with the `createTree` calls

Reply:
Do we want `ignoreDeletionFailures` to be `change` or `changeS` level scope?
If change level we discard the change on failure and if changes we discard the entire submission.

I believe the first option makes more sense.